### PR TITLE
apply clang-format 10 changes

### PR DESCRIPTION
**Pull request by v4hn**
Looks like migrating MoveIt (and the related change in moveit_ci)
triggered the migration for this repository as well (observed [here](https://travis-ci.com/github/ros-planning/moveit_task_constructor/jobs/375505515#L569))

I only added `SplitEmptyRecord: false` to our current config and the results
look acceptable to me.

### CODE REVIEW

**Comment by v4hn** [comment]
Anything you disagree with @rhaschke ?

File: core/src/stages/compute_ik.cpp [hunk 1]
```diff
@@ -354,8 +354,9 @@ void ComputeIK::compute() {
 	double min_solution_distance = props.get<double>("min_solution_distance");
 
 	IKSolutions ik_solutions;
-	auto is_valid = [sandbox_scene, ignore_collisions, min_solution_distance, &ik_solutions](
-	    robot_state::RobotState* state, const robot_model::JointModelGroup* jmg, const double* joint_positions) {
+	auto is_valid = [sandbox_scene, ignore_collisions, min_solution_distance,
+	                 &ik_solutions](robot_state::RobotState* state, const robot_model::JointModelGroup* jmg,
+	                                const double* joint_positions) {
```
Comment (rhaschke) [inline core/src/stages/compute_ik.cpp:359]: This one looks worse than before.

File: visualization/visualization_tools/src/task_solution_visualization.cpp [hunk 2]
```diff
@@ -404,9 +406,8 @@ void TaskSolutionVisualization::update(float wall_dt, float ros_dt) {
 			current_state_time_ = 0.0;
 			trail_scene_node_->setVisible(false);
 		} else if (tm < 0.0) {  // using realtime: skip to next waypoint based on elapsed display time
-			while (current_state_ < max_state_index &&
-			       (tm = displaying_solution_->getWayPointDurationFromPrevious(current_state_ + 1)) <
-			           current_state_time_) {
+			while (current_state_ < max_state_index && (tm = displaying_solution_->getWayPointDurationFromPrevious(
+			                                                current_state_ + 1)) < current_state_time_) {
```
Comment (rhaschke) [inline visualization/visualization_tools/src/task_solution_visualization.cpp:410]: This as well.

**Review by rhaschke — COMMENTED** [review]
Most changes are reasonable, but some are worse than before.
Give me a moment to try improving this.

**Review by rhaschke — APPROVED** [review]
Didn't find tweaks to improve the two places.

**Pull request merged**